operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -380,7 +380,7 @@ metadata:
       ["Red Hat AMQ", "Red Hat Application Foundations"]
     operators.operatorframework.io/internal-objects: |-
       ["strimzipodsets.core.strimzi.io"]
-    olm.skipRange: '>=2.4.0-0 <2.5.2-5'
+    olm.skipRange: '>=2.2.0-0 <2.5.2-6'
```

Comment:
This is fine I think, but technically not needed with the change made in the automation script.